contrib/tmpfiles.d/acmed.conf

```diff
@@ -0,0 +1,16 @@
+####
+# acmed:
+# running as sandboxed daemon (UID/GID) needs Read/Write access
+# preset runtime environment, if not restricted via systemd unit
+###
+
+#Type Path                               Mode User  Group Age Argument
+d     /run/acmed                         0755 acmed acmed -   -
+f     /run/acmed/acmed.pid               0644 acmed acmed -   -
+
+d     /var/lib/acmed                     0755 acmed acmed -   -
+d     /var/lib/acmed/accounts            0700 acmed acmed -   -
+d     /var/lib/acmed/certs               0755 acmed acmed -   -
+d     /var/lib/acmed/domains             0755 acmed acmed -   -
+
+d     /etc/nginx/conf.d			 0775 root  acmed -   -
```

Comment:
This a no-go.
Installing ACMEd must not create such a directory on systems that do not have nginx installed.

Reply:
You are absolutely right. We shouldn't infere on system directories of other packages. Probably best, to come up with a solution that is parsing on installation using the OS dependent packager. Haven't made up my mind, on how to implement this.
Maybe it is best, to just create the templates in a domain specific subdir of acmed and let the admin include this templates in a manual fashion. This can be explained in the documentation.

Will fix the systemd tempfile.